.github/workflows/multi-cluster-config.yaml

```diff
@@ -13,13 +13,13 @@ jobs:
         name: "Create 1st k3d Cluster"
         with:
           cluster-name: "test-cluster-1"
-          args: --config=.github/workflows/assets/1.yaml
+          args: --config=.github/workflows/assets/1.yaml # test: k3s:v1.20.4-k3s1
```

Comment:
@kuritka do you really want to mention an explicit tag in the comment? You will have to manually sync them up with https://github.com/AbsaOSS/k3d-action/pull/26/files#diff-7d4a88946806380cd1533fc58c7dabd83bd8bc8b4c1311edbd2cc167b9d62463R3  all the time